core/domain/fs_services.py

```diff
@@ -21,6 +21,12 @@
 
 
 def get_exploration_file_system_class():
+    """Returns ExplorationFileSystem class to the client if DEV_MODE is True.
+    Otherwise, returns GcsFileSystem.
+
+    Returns:
+        ExplorationFileSystem|GcsFileSystem. The correct file system class.
```

Comment:
This should be:

    class. The correct file system class (either ExplorationFileSystem or GcsFileSystem).

This is because you're returning the class itself, and not an instance of ExplorationFileSystem.

Reply:
Done